client/packages/common/src/api/GqlContext.tsx

```diff
@@ -25,7 +25,7 @@ const permissionExceptions = [
   'invoiceCounts',
   'itemCounts',
   'requisitionCounts',
-  'temperatureBreaches',
+  'temperatureNotifications',
```

Comment:
Nit: a comment what permissionExceptions is used for would be good, i.e. that it contains endpoints which are optional in a sense that a user might not have permissions for them but calling them will not redirect the user to the login page but just gives a notficiation about missing permissions (?)